index.js

```diff
@@ -49,11 +49,23 @@ const cpFileAsync = async (source, destination, options, progressEmitter) => {
 	}
 };
 
+const changeCurrentWorkingDirectory = (cwd) => {
+	try {
+		process.chdir(cwd);
```

Comment:
You should never call `process.chdir` in packages. It applies to everything running in the process.

The correct approach is to transform all the paths.